src/components/header/components/Navigation.jsx

```diff
@@ -70,6 +83,35 @@ const Navigation = ({ isMenuOpen, setIsMenuOpen, mobile, tablet }) => {
   const [navigationState, setNavigationState] =
     useRecoilState(navigationStateAtom);
 
+  const cartIcon = useSetRecoilState(cartClick);
+  // const cartIcon = useRef();
```

Comment:
Still seeing a comment here, why leave it in?

Reply:
✅